File: src/main/java/com/tsquare/speakfriend/entity/AccountEntity.java
package com.tsquare.speakfriend.entity;

import java.sql.ResultSet;
import java.sql.SQLException;

public class AccountEntity extends Entity {
    AccountEntity(ResultSet resultSet) throws SQLException {
        super(resultSet);
    }

    public int getId() {
        return (int) props.get("id");
    }

    public String getName() {
        return (String) props.get("name");
    }

    public String getUser() {
        return (String) props.get("user");
    }

    public String getPass() {
        return (String) props.get("pass");
    }

    public String getUrl() {
        return (String) props.get("url");
    }

    public String getNotes() {
        return (String) props.get("notes");
    }
}

File: src/main/kotlin/com/tsquare/speakfriend/database/account/AccountEntity.kt
package com.tsquare.speakfriend.database.account

import com.tsquare.speakfriend.database.tables.Accounts
import org.jetbrains.exposed.dao.Entity
import org.jetbrains.exposed.dao.EntityClass
import org.jetbrains.exposed.dao.EntityID

class AccountEntity(id: EntityID<Int>): Entity<Int>(id) {
    companion object: EntityClass<Int, AccountEntity>(Accounts)

    var userId by Accounts.userId
    var name by Accounts.name
    var user by Accounts.user
    var pass by Accounts.pass
    var url by Accounts.url
    var notes by Accounts.notes
}
